tools/phenomenal/w4m/update-tools

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I run the update-tools script locally on my host, and then commit the changes in git repos. We could move this script outside the tools folder, of course. Just rename it something like update-w4m-tools, since it runs only for w4m tools for the moment.